src/task/task.controller.ts

```
@@ -39,7 +39,7 @@ export class TaskController {
     @ApiResponse({ status: 200, description: 'Task successfully edited' })
     @ApiResponse({ status: 401, description: 'Unauthorized' })
     @ApiResponse({ status: 403, description: 'Forbidden.' })
-    @ApiResponse({ status: 500, description: 'Task ID does not exist'})
+    @ApiResponse({ status: 500, description: 'Task ID does not exist' })
     @UseGuards(JwtAuthGuard, AdminGuard)
     editTaskAdmin(@Param('id') id: number, @Body() task: UpdateTaskDto): Promise<Object> {
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)